ansible/roles/pre-ansible/tasks/main.yml

```diff
@@ -1,31 +1,29 @@
 ---
+- name: get hostname facts
+  raw: "hostname"
+  register: hostname
+
+- name: add hostname to /etc/hosts
```

Comment:
I guess it means that you dont delete the file but add it as a first line, correct?